docker/Makefile

```diff
@@ -24,14 +24,7 @@ endif
 .PHONY: unit-tests
 unit-tests: build
 	docker-compose build test-php
-ifeq ("$(TEAMCITY_VERSION)",$())
-	# developer machine
 	docker-compose run test-php
-else
-	# teamcity CI
-	docker-compose run -e WEBSITE=languageforge.org --name unittests test-php
-	docker cp unittests:/var/www/PhpUnitTests.xml ..
```

Comment:
note to self: this line is what we need to use for GA to recognize the PHP Unit test results in the test formatter

Reply:
There is also discussion about this here: https://github.com/sillsdev/web-languageforge/pull/1258#discussion_r760716331